playbooks/aws/provision.yml

```diff
@@ -27,3 +27,21 @@
     - create_users
   tags:
     - openshift_deploy_cluster
+
+- hosts: openshift_nodes:&openshift_cluster_{{ hostvars.localhost.cluster_group_name }}
+  tasks:
+  - name: Prepare all nodes for Istio
+    include_role:
+      name: host_prep_istio
+    when: addon_istio
```

Comment:
If you convert the way you specify this to:

```yaml
addons:
  - istio
```

Then you would conver this to:

```yaml
when: "'istio' in addons"

Reply:
I added this to my PR to your branch.